test/e2e/specs/specs-playwright/wp-likes-spec.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+import {
+	BrowserHelper,
+	DataHelper,
+	LoginFlow,
+	NewPostFlow,
+	GutenbergEditorPage,
+	LikesComponent,
+} from '@automattic/calypso-e2e';
+
+/**
+ * Constants
+ */
+const mochaTimeOut = config.get( 'mochaTimeoutMS' );
+const host = DataHelper.getJetpackHost();
+const viewportName = BrowserHelper.getViewportName();
+const quote =
+	'The foolish man seeks happiness in the distance. The wise grows it under his feet.\n— James Oppenheim';
+
+describe( `[${ host }] Likes: (${ viewportName }) @parallel`, function () {
+	this.timeout( mochaTimeOut );
+
+	describe( 'New post', function () {
+		let gEditor;
+		let likesComponent;
+
+		before( 'Can log in', async function () {
+			const loginFlow = new LoginFlow( this.page, 'gutenbergSimpleSiteUser' );
+			await loginFlow.login();
+		} );
+
+		it( 'Start new post', async function () {
+			const newPostFlow = new NewPostFlow( this.page );
+			await newPostFlow.newPostFromNavbar();
+		} );
+
+		it( 'Enter post title', async function () {
+			gEditor = await GutenbergEditorPage.Expect( this.page );
+			const title = DataHelper.randomPhrase();
+			await gEditor.enterTitle( title );
+			await gEditor.enterText( quote );
```

Comment:
Should we be checking if the title and quote are actually rendered in the editor canvas?

Reply:
I think it would be good to have two additional `GutenbergEditorPage` methods for verification: `getTitle` and `getText`. Then we could verify their presence and add proper assertions here.

Reply:
Sure, should not be difficult to do. Though Playwright would [throw an error if any of these fail](https://playwright.dev/docs/api/class-page#pagefillselector-value-options):
- selector cannot be found.
- element is attached to DOM.
- is visible.
- is enabled.
- is editable.

Only then is the element focused and filled.

I think adding a `frame.isVisible(selector)` should take care of checking on our end if text fill was successful.

Reply:
Implemented `getText` and `getTitle` functions to permit verification of if the title and/or text was entered correctly.

This involved some investigation and frustration, surprisingly. It is noted in the comments in code, but the gist is:
- Playwright's `fill` function ignores newline characters, while `type` will enter as a human would, it is slow. I had to split the text up by newline, generate a list of promises to fill them one line at a time.
- in the process of getting the list of paragraph blocks to piece back together the text that is on page, I encountered the U+FEFF character which is present on empty paragraph blocks as a placeholder. This muddled up the string I was piecing back together.

I was able to overcome the issues for this test case, but we will see if my solution can stand up to other test cases.